pkg/model/bundle.go

```diff
@@ -195,5 +191,5 @@ func IsGeneratedFile(file string) bool {
 func init() {
 	metaRe = regexp.MustCompile(`^\.datamon/(.*)\.yaml$`)
 	flRe = regexp.MustCompile(`^(.*)-` + bundleFilesIndexPrefix + `(.*)$`)
-	genFileRe = regexp.MustCompile("^.datamon/*|^/.datamon/*|^/.datamon$|^.datamon$|^./.datamon/*|^./.datamon$")
+	genFileRe = regexp.MustCompile(`^\.datamon/.*|^/\.datamon/.*|^/\.datamon$|^\.datamon$|^\./\.datamon/.*|^\./\.datamon$|^\.?/?\.conflicts(/.*|$)|^\.?/?\.checkpoints(/.*|$)`)
```

Comment:
recall:  `.conflicts` and `.checkpoints` are part of the diamond conflict resolution mechanism described in the design document